tensorflow/core/platform/file_system.cc

```diff
@@ -436,4 +436,13 @@ string FileSystem::CreateURI(StringPiece scheme, StringPiece host,
   return strings::StrCat(scheme, "://", host, path);
 }
 
+string FileSystem::DecodeTransaction(const TransactionToken* token){
+  if(token){
+    std::stringstream oss;
+    oss<<"Token= "<<token->token<<", Owner="<<token->owner;
+    return oss.str();
+  }
```

Comment:
Let's use `StrCat` (https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/core/platform/strcat.h;l=147-176;drc=16b5232a803abcb55a30fa8326edbe279aca2a1e)

Reply:
Hi, @mihaimaruseac,

This is what I intended to use it first but AFAIK StrCat can not handle void* without some casting. Am I  wrong?

Reply:
Let's add the needed cast/`ToString` method.

Also, this should return `std::string`, not `string` (will probably trip the linter upon import, even if it compiles now)

Reply:
@mihaimaruseac 
I added a TODO message for switching to StrCat when void* is supported and changed the signature to std::string.